### ISSUE TITLE
Endianness of <switches default=>

### ISSUE THREAD

**Issue by gyurco** [sebdel/mra-tools-c#90]
On MiSTer, it looks like little endian:
https://github.com/MiSTer-devel/Main_MiSTer/blob/master/support/arcade/mra_loader.cpp#L527
But in the MRA tool, it's big endian:
https://github.com/gyurco/mra-tools-c/blob/master/src/mra.c#L224

@jotego: can you confirm it?

**Comment by jotego** [comment]
Do you mean that the default value of DIP switches is read in a different way for MiSTer and MiST?
I don't think I have noticed that problem. But it's true that I normally have all FF's
I'll check it with a game a report back.

**Comment by gyurco** [comment]
Yes, of course when all is FF, than it doesn't matter. But in 1942 for example, it does.
